src/MongoDB.Driver.Core/Core/Operations/FindOneAndReplaceOperation.cs

```diff
@@ -178,6 +194,7 @@ internal override BsonDocument CreateCommand(ICoreSessionHandle session, Connect
                 { "writeConcern", writeConcern, writeConcern != null },
                 { "bypassDocumentValidation", () => _bypassDocumentValidation.Value, _bypassDocumentValidation.HasValue && Feature.BypassDocumentValidation.IsSupported(serverVersion) },
                 { "collation", () => Collation.ToBsonDocument(), Collation != null },
+                { "hint", () => Hint, Hint != null },
```

Comment:
We normally use `_hint` (instead of `Hint`) when we are in the same class.

Reply:
Done.